packages/ccache.rb

```diff
@@ -53,7 +54,7 @@ def self.install
       FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
       @ccacheenv = <<~CCACHEEOF
         # ccache configuration
-        if [ "\$PATH" -eq "*ccache/bin*" ]; then
+        if ! [[ \$PATH == *"ccache/bin"* ]]; then
```

Comment:
`unless [[ \$PATH == *"ccache/bin"* ]]; then`?

Reply:
That's bash shell code, not ruby!

Reply:
Oh, my bad.